lib/ansible/modules/cloud/docker/docker_image.py

```diff
@@ -510,6 +516,7 @@ def build_image(self):
             tag=self.name,
             rm=self.rm,
             nocache=self.nocache,
+            network_mode=self.network,
```

Comment:
Support for `network_mode` has only been added in docker-py 2.4.0 (see [changelog](https://docker-py.readthedocs.io/en/stable/change-log.html)), while this module also supports docker-py 1.7.0.

You should add some version check similar to the checks in docker_container (search for `HAS_CPUSET_MEMS_OPT` as an example) to avoid problems because of that.

Reply:
You also need API version 1.25 (the module requires 1.20), so better search for `HAS_INIT_OPT` in `docker_container`.